docs/content/migration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Migrate from Primer CSS
+---
+
+Many Primer CSS use cases are supported by Primer ViewComponents.
+
+When migrating from Primer CSS classes to ViewComponents, use this mapping to
+help guide your implementation. This list includes components currently in
+stable status.
+
+| Primer CSS Class | ViewComponent |
+|------------------|---------------|
+| `State`                   | `Primer::StateComponent`              |
```

Comment:
What do you think about having both of these columns be links to the respective libraries' page for each pattern?

Reply:
Yup, can do! The bigger issue is some of these components are not actually in stable at the moment...